src/components/Calendar.js

```diff
@@ -87,29 +121,51 @@ class Calendar extends React.Component {
     return weeks;
   };
 
-  _renderMonthTable = (styles) => {
+  _renderMonthTable = styles => {
```

Comment:
A nice follow up here next would be to refactor these into components.

Reply:
👍